integrated_channels/moodle/models.py

```diff
@@ -230,7 +230,10 @@ def is_valid(self):
         incorrect_items = {'incorrect': []}
         if not self.moodle_base_url:
             missing_items.get('missing').append('moodle_base_url')
-        if not self.token and not (self.username and self.password):
+        if (not self.token and not (self.username and self.password)) or (
+            not self.decrypted_token
+            and not (self.decrypted_username and self.decrypted_password)
+        ):
```

Comment:
shouldn't we directly just check the decrypted values here now that we are completely replacing the fields?

Reply:
@sameenfatima78 as both unencrypted and encrypted values exist for now 
therefore i was making the check using both.

Reply:
We should merge the PR with new columns now. The old can stay in DB but this pr should handle entirely replacing the fields with encrypted columns everywhere.